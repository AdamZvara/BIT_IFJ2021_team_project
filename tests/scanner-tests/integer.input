123
0456
00000000007
90002
