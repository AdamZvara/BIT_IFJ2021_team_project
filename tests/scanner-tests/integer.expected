TOK_INT : 123
TOK_INT : 456
TOK_INT : 7
TOK_INT : 90002
